frank-doc-doclet/src/main/java/org/frankframework/frankdoc/Utils.java

```diff
@@ -270,7 +270,7 @@ private static String getLinkReplacement(String linkBody) {
 	}
 
 	public static String replaceClassFieldValue(String text, FrankClass context) throws FrankDocException {
-		return replacePattern(text, JAVADOC_VALUE_START_DELIMITER, JAVADOC_SUBSTITUTION_PATTERN_STOP_DELIMITER, s -> getClassFieldValueReplacement(s, context));
+		return replacePattern(text, JAVADOC_VALUE_START_DELIMITER, JAVADOC_SUBSTITUTION_PATTERN_STOP_DELIMITER, s -> "<code>" + getClassFieldValueReplacement(s, context) + "</code>");
```

Comment:
Ik herinner me niet meer hoe deze code precies werkt, dus neem niet zomaar aan wat ik hier opmerk. Is het een idee om de `<code>` en `</code>` tags op ge nemen in functie `getClassFieldValueReplacement()`?